configs/test_local.config

```diff
@@ -0,0 +1,31 @@
+workDir = "work/"
+
+params {
+    outdir = "./test_result"
+}
+
+docker {
+    enabled       = true
+    registry      = 'quay.io'
+    runOptions    = "--platform linux/amd64"
+}
+
+executor {
+    name          = "local"
+    cpus          = 2
+    memory        = '1 GB'
+}
+
+singularity {
+    enabled = false
+    autoMounts = true
+    cacheDir = "/hps/nobackup/rdf/metagenomics/service-team/singularity-cache/"
```

Comment:
![image](https://github.com/user-attachments/assets/4ec726cd-d204-485e-b587-410adb915e4e)

test_local? I would assume local means you own laptop

Reply:
Yes, I disabled the singularity and forgot to remove the rest 😅
Fixed in commit [remove confusing singularity params](https://github.com/EBI-Metagenomics/taxdb_generation_nf/pull/1/commits/db04017fe5c5f57ffc32c84f3f122579a60c4cb2)